### DISCUSSION TITLE
Tutorial & Samples

### DISCUSSION THREAD

**Discussion by Bartos-Pol** [Live-Charts/LiveCharts2#83] [Q&A] [answered]
Hi,
Is there a tutorial for new verion LiveCharts2? 
How to generate Labels and Values from linq ?

**Comment by beto-rodriguez** [comment, marked as answer]
Today there is only the [docs folder](https://github.com/beto-rodriguez/LiveCharts2/tree/master/docs), this is the base of the documentation that the new web site will use to build the docs, and there is the [samples folder](https://github.com/beto-rodriguez/LiveCharts2/tree/master/samples) where you can find many built samples.

It will be easier to browse once the new site is published, I am getting ready the last details to publish it.

> How to generate Labels and Values from linq ?

What do you mean?

**Reply by Bartos-Pol** [reply]
I keep my fingers crossed for this project :) 
Maybe you have smal sample how to generate ISeries from eg,
var sample = from s on List
                      group s by s.Date into g
                     select new ...... 
it's just a suggestion ;)
